style.css

```diff
@@ -79,3 +92,252 @@ h1 {
   margin: 2%;
   padding-left: 10px;
 }
+
+.global {
+  width: 100%;
+  height: 2788px;
+  top: 718px;
+  padding: 114px 24px 114px 24px;
+  gap: 88px;
+  background-color: #dfe1e6;
+}
+
+.image {
+  width: 287px;
+  height: 233px;
+  border-radius: 8px;
+}
+
+.presentation {
+  width: 295px;
+  height: 288px;
+  gap: 12px;
+}
+
+ul {
+  list-style: none;
+}
+
+.project_title {
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+}
+
+.frame {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 236px;
+  height: 32px;
+  gap: 12px;
+}
+
+.dot {
+  border-radius: 50%;
+  background: #c1c7d0;
+  width: 8px;
+  height: 8px;
+}
+
+.client {
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+}
+
+.list {
+  font-family: Poppins, sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #7a869a;
+}
+
+.primary {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  padding-bottom: 30px;
+}
+
+.tag {
+  width: 214px;
+  height: 24px;
+  gap: 8px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 30px;
+}
+
+.bouton {
+  font-family: Poppins, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #6070ff;
+}
+
+.b1 {
+  width: 57px;
+  height: 24px;
+  padding: 4px 12px 4px 12px;
+  border-radius: 8px;
+  background-color: #ebebff;
+}
+
+.b2 {
+  width: 49px;
+  height: 24px;
+  padding: 4px 12px 4px 12px;
+  border-radius: 8px;
+  background-color: #ebebff;
+}
+
+.b3 {
+  width: 92px;
+  height: 24px;
+  padding: 4px 12px 4px 12px;
+  border-radius: 8px;
+  background-color: #ebebff;
+}
+
+.btn {
+  width: 126px;
+  height: 48px;
+  top: 12px;
+  padding: 12px;
+  border-radius: 8px;
+  gap: 10px;
+  margin: 12px 0;
+}
```

Comment:
- [ ] Good job in implementing the changes requested by the Previous Reviewer, However, as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=ZwtntAgqTPwDdRBG-0) and as the Previous Reviewer mentioned above, please ensure that your design matches the given design. matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is the Screenshot showing the difference: :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![Student width](https://github.com/medijdir/mobileportfolio/assets/43528594/014a267a-6708-4d68-aa8a-6bed3c6bdf59)

</td>
      <td>

![figma Design width](https://github.com/medijdir/mobileportfolio/assets/43528594/d95c5edb-87bc-4424-a624-86648356fcb1)

</td>
    </tr>
  </table>

Reply:
hi @Alusp and @Cabraham1 ,
Thanks for your help and your good work, please I have made it my better to stick to the Figma design. I have fixed the last print screen issue and this is my last review that is why kindly I need your help to tell me if there are other issues to do it.

Reply:
Hi @medijdir, You still have 3 extra reviews to use. These are more than enough to get your project approved if you commit to implementing the changes requested. 😄